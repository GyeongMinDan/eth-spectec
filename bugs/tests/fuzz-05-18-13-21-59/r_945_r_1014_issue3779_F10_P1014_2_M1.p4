list<_> f() {
}
control c(out bool b)() {
  apply {
  }
}
