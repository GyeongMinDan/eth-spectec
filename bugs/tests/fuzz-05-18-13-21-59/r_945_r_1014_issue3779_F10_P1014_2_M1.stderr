likely_hits/r_945_r_1014_issue3779_F10_P1014_2_M1.p4(3): [--Wwarn=unused] warning: Control c is not used; removing
control c(out bool b)() {
        ^
[--Wwarn=missing] warning: Program does not contain a `main' module
